src/doc/src/reference/registry-index.md

```diff
@@ -199,6 +199,9 @@ explaining the format of the entry.
     "features2": {
         "serde": ["dep:serde", "chrono?/serde"]
     }
+    // Required version of Rust (optional).
```

Comment:
```suggestion
    // Minimum required version of Rust (optional).
```

Reply:
Or be consistent with the current doc — The minimal supported Rust version.

https://doc.rust-lang.org/nightly/cargo/reference/manifest.htm

Reply:
yeah, that sounds even better :)